Dockerfile.dapper

```diff
@@ -74,7 +86,11 @@ RUN cd /go/src/github.com/ && \
 RUN pip3 install grpcio==1.22.0 grpcio_tools==1.22.0
 
 # buf
-RUN wget -O /usr/bin/buf https://github.com/bufbuild/buf/releases/download/v0.16.0/buf-Linux-x86_64 && chmod +x /usr/bin/buf
+ENV GO111MODULE=on
+RUN go get \
+  github.com/bufbuild/buf/cmd/buf \
+  github.com/bufbuild/buf/cmd/protoc-gen-buf-check-breaking \
+  github.com/bufbuild/buf/cmd/protoc-gen-buf-check-lint
```

Comment:
since these are external dependencies do we want to tag these to a specific version?

Reply:
Good point. I will fix it. Thanks.